pglookout/cluster_monitor.py

```diff
@@ -177,8 +177,11 @@ def _standby_status_query(self, instance, db_conn):
                 # This is only run on masters to create txid traffic every db_poll_interval
                 phase = "updating transaction on"
                 self.log.debug("%s %r", phase, instance)
-                c.execute("SELECT txid_current()")
+                # We simulate replay_location with the results of pg_current_xlog_location on master
+                c.execute("SELECT txid_current(), pg_current_xlog_location() AS pg_last_xlog_replay_location")
```

Comment:
txid_current() is presumably called to force a new transaction to be written, so this is just missing a comment